packages/google_maps_flutter/google_maps_flutter_android/lib/src/google_maps_flutter_android.dart

```diff
@@ -625,6 +631,22 @@ class GoogleMapsFlutterAndroid extends GoogleMapsFlutterPlatform {
     GoogleMapsInspectorPlatform.instance =
         GoogleMapsInspectorAndroid((int mapId) => _channel(mapId));
   }
+
+  /// Intializes Google Maps SDK for Android with preferred renderer
+  ///
+  /// The returned [Future] completes after renderer has been initialized.
+  /// Initialized renderer type is returned
+  /// Supported on Android only.
+  @override
+  Future<String> initializeWithRenderer(String renderer) async {
```

Comment:
This should not be a string-based API; this should be what `initializeGoogleMapsWithAndroidMapRenderer` currently is (and that separate method doesn't need to exist).